Makefile.frag

```diff
@@ -0,0 +1,77 @@
+
+INSTALL_DIR=$(shell pwd)/build/install
+
+crt_libs=$(INSTALL_DIR)/lib/libaws-c-common.a \
+	$(INSTALL_DIR)/lib/libaws-c-cal.a \
+	$(INSTALL_DIR)/lib/libaws-c-io.a \
+	$(INSTALL_DIR)/lib/libaws-c-compression.a \
+	$(INSTALL_DIR)/lib/libaws-c-http.a \
+	$(INSTALL_DIR)/lib/libaws-c-auth.a \
+	$(INSTALL_DIR)/lib/libs2n.a
+
+crt-static: $(crt_libs) $(INSTALL_DIR)/lib/libcrypto.a
+
+build/deps/openssl:
+	@echo Fetching libcrypto source
+	git clone --single-branch --branch OpenSSL_1_1_1 https://github.com/openssl/openssl.git build/deps/openssl
```

Comment:
trivial: `--depth 1` to speed things up even more?

Reply:
shouldn't the branch be [OpenSSL_1_1_1-stable](https://github.com/openssl/openssl/tree/OpenSSL_1_1_1-stable)?

Reply:
Good catch!